### PR TITLE
make status checks configurable from env vars + use shorter replication interval for testing

### PR DESCRIPTION
**Pull request by jrhizor**
We spend quite a lot of time in Kubernetes integration testing just waiting for status checks. Here I'm allowing the values to be configurable in the environment and then using those for both of the Kuberentes acceptance tests, hopefully increasing the speed of the tests.

I split this into two commits so it'd be easier to review but also so we can get a sense of the speed up. I think this will help on local and CI iteration when running the acceptance tests. (EDIT: I forgot to push them individually, so I added two reverts so we can actually compare)

@pmossman I don't really have a use case for configuring the other values outside of replication, but I just figured it would make sense to have these easily configurable.

### CODE REVIEW

**Comment by pmossman** [comment]
Haha so we had some discussion around whether or not these status check intervals should be configurable via env-vars, thread here: https://docs.google.com/document/d/1w9UBa1xTQcpJYLjmWi8zMLYN3vZ6DhDVyTxFIM5hgxk/edit?disco=AAAAVLvHHqI

We didn't consider acceptance testing as a use case for wanting to configure shorter intervals, so I think this makes sense. @cgardens @davinchia FYI in case you have a stronger opinion than I do

**Review by pmossman — APPROVED** [review]
See my comment about the hardcoding discussion, I don't think the opinions expressed were strong so I'm comfortable moving forward with this to speed up our tests. I kind of doubt we'll see much/any usage from the configuration of non-replication-worker intervals but it doesn't hurt for the sake of consistency

**Review by davinchia — APPROVED** [review]
haha yes, not strong opinions. this is good!

File: airbyte-config/models/src/main/java/io/airbyte/config/Configs.java [hunk 1]
```diff
@@ -293,6 +294,41 @@
    */
   String getJobKubeNamespace();
 
+  /**
+   * Define the interval for checking for a Kubernetes pod status for a worker of an unspecified type.
+   *
+   * In seconds if specified by environment variable.
+   */
+  Duration getDefaultWorkerStatusCheckInterval();
```
[the remark this answers is not in the document]
Comment (jrhizor) [reply airbyte-config/models/src/main/java/io/airbyte/config/Configs.java:302]: oops missed this.

Added in https://github.com/airbytehq/airbyte/pull/10390

**Pull request merged**